internal/manifests/internal/gateway/gateway-tenants.yaml

```diff
@@ -17,24 +17,22 @@ tenants:
     issuerCAPath: {{ $secret.IssuerCAPath }}
     {{- end -}}
     {{- end -}}
-    {{- end -}}
-    {{ print "\n" }}
```

Comment:
Was it working without `{{ print "\n" }}`? It was generating wrong yaml previously when we omit this.

Reply:
Yes, you need to tune when to use the `-` in starting blocks `{{` and end blocks `}}`.